### PR TITLE
Home page layout

### PR DESCRIPTION
**Pull request by DeviPriya-Kolla**
Designed a basic home page layout with header bar and footer bar.

### CODE REVIEW

**Comment by JoshBudzynski** [comment]
Adding a portion of the Glossary, and drop down menu. These are not linked

File: bones/mandible.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```
Comment (Balijah) [inline bones/mandible.html:1]: I feel like the use of spaces and overall formatting could be cleaned up a little bit. Could be easier on the eyes but this is just personal preference.

File: frontend/styles.css [hunk 2]
```diff
@@ -0,0 +1,160 @@
+:root {
```
Comment (Balijah) [inline frontend/styles.css:1]: Additional comments could never hurt!

**Comment by kate-holdener** [comment]
> Partially fixes issue #4
> 
> Added a glossary drop down menu, with important bones names. Along with select bones with separate HTML pages adding detail to that bone.
> 
> This was added to allow the team to have the same knowledge base.
> 
> I wanted to add a HTML page for every bone however, I fell short where the link of HTML pages, they aren't linked correctly. Which took up much of my time so I wasn't able to add a page for every bone.

Please use:
fixes #4 
To automatically connect the PR with the issue.

**Pull request closed without merging**